planner/core/rule_join_reorder_greedy.go

```diff
@@ -54,6 +55,8 @@ func (s *joinReorderGreedySolver) solve(joinNodePlans []LogicalPlan, tracer *joi
 		})
 		tracer.appendLogicalJoinCost(node, cost)
 	}
+
+	// Sort plans by cost and join order
```

Comment:
How to manifest `sort plans by join order`?

Reply:
done